openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -83,17 +125,51 @@ <h5 class="submission__answer__part__text__title">{% trans "The prompt for this
                                 </article>
 
 
+                                {% if team_name %}
+                                  <div id='team_name_and_users'>
+                                      {% trans "You are on team " %}
+                                      <a href="{{team_url}}">{{team_name}}</a>.
+                                      {% trans "Team Members: " %}
+                                      <strong class="emphasis">{{team_usernames|join:", "}}.</strong>
+                                      {% if team_members_with_external_submissions %}
+                                        <br />
+                                        <strong class="emphasis">
+                                          {% blocktrans %}
+                                              {{team_members_with_external_submissions}}
+                                              have/has already submitted a response to this assignment with another team,
+                                              and will not be a part of your team's submission or assignment grade.
+                                          {% endblocktrans %}
+                                        </strong>
+                                      {% endif %}
+                                  </div>
+                                {% endif %}
+
                                 {% if text_response %}
                                 <div class="field field--textarea submission__answer__part__text">
                                     <div class="submission__answer__part__text">
                                         <h5 id="submission__answer__part__text__title__{{ forloop.counter }}__{{ xblock_id }}"
                                             class="submission__answer__part__text__title">
+                                          {% if team_name %}
+                                            {% trans "Team Response " %}
+                                          {% else %}
+                                            {% trans "Your Response " %}
+                                          {% endif %}
+
                                           {% if text_response == "required" %}
-                                            {% trans "Your response (required)" %}
+                                            {% trans "(Required)" %}
                                           {% elif text_response == "optional" %}
-                                            {% trans "Your response (optional)" %}
+                                            {% trans "(Optional)" %}
                                           {% endif %}
                                         </h5>
+                                        {% if team_name %}
+                                          {% blocktrans %}
+                                              Teams should designate one team member to submit a response on behalf of the
```

Comment:
This comment makes my head hurt...Have we asked UX for a better verbiage if at all possible?

Reply:
This came from UX and product. But I agree, it is a bit verbose.